app/assets/javascripts/discourse/app/lib/sidebar/common/community-section/everything-section-link.js

```diff
@@ -63,6 +63,14 @@ export default class EverythingSectionLink extends BaseSectionLink {
   }
 
   get route() {
+    if (this.currentUser?.sidebarListDestination === "unread_new") {
```

Comment:
Instead of duplicating `unread_new` all around, I think it'll be helpful if we extract it to a constant and export it.